doc/01_Getting_Started/00_Installation/01_Webserver_Installation.md

```diff
@@ -80,8 +80,33 @@ The installer writes a log in `var/log` which contains any errors encountered du
 ## 5. Maintenance Cron Job
 
 Maintenance tasks are handled with Symfony Messenger. The `pimcore:maintenance` command will add the maintenance
-messages to the bus and run them afterward immediately from the queue. However, it is  recommended to set up independent workers that process the queues by running `bin/console messenger:consume pimcore_core pimcore_maintenance pimcore_image_optimize pimcore_asset_update` (using e.g.
-`Supervisor`).
+messages to the bus. `pimcore:maintenance` however, will not process the maintenance messages itself. Instead,
+independent workers are required to consume these messages from the queues and to actually perform maintenance tasks.
+In a typical pimcore installation you should set up workers for these transports:
+
+| # | Transport name                 | Purpose                                                                                       |
+|---|--------------------------------|-----------------------------------------------------------------------------------------------|
+| 1 | pimcore_core                   | Core background tasks such as converting videos, rendering asset previews, etc.               |
+| 2 | pimcore_maintenance            | [Maintenance tasks](../../20_Extending_Pimcore/16_Maintenance_Tasks.md)                       |
+| 3 | pimcore_scheduled_tasks        | [Scheduled tasks](../../18_Tools_and_Features/03_Scheduling.md)                               |
+| 4 | pimcore_image_optimize         | Apply optimizations to image assets                                                           |
+| 5 | pimcore_asset_update           | Updates of asset metadata                                                                     |
+| 6 | pimcore_search_backend_message | Updates the index for pimcore backend search. Only if you install `SimpleBackendSearchBundle` |
+
+Beware, that the actual transports depend on which core bundles you chose to install, so you will need to adjust the list
+in accordance with your installation. Use `bin/console debug:messenger` to list all available transports.
+
+Consuming messages from the queues is performed with the `messenger:consume` command, i.e.: `bin/console messenger:consume pimcore_core ...`
+You might need to spawn multiple workers, each consuming from a different queues to avoid blocking message processing by long-running tasks,
```

Comment:
```suggestion
You might need to spawn multiple workers, each consuming from a different queue to avoid blocking message processing by long-running tasks,
```